tests/test_smoke.py

```diff
@@ -678,6 +680,35 @@ def test_gcp_storage_mounts():
         run_one_test(test)
 
 
+@pytest.mark.cloudflare
+def test_cloudflare_storage_mounts(generic_cloud: str):
```

Comment:
`pytest tests/test_smoke.py::test_cloudflare_storage_mounts --cloudflare` fails with:

```
ValueError: Cloud cloudflare is not a valid cloud among ['aws', 'azure', 'gcp', 'lambda', 'local']
Failed.
Reason: sky launch -y -c t-cloudflare-storab5c-a56f-d6 --cloud cloudflare /var/folders/98/hhq8wrtx6y13196q61xphjsm0000gn/T/tmp5ay5w1vi.yaml
Log: less /var/folders/98/hhq8wrtx6y13196q61xphjsm0000gn/T/cloudflare_storage_mounts-sfyxxvg9.log
```

We need to fix this by setting the launch cloud to default_cloud_to_run[0] when `--cloudflare` is specified. Might need to look at [def _generic_cloud in conftest.py](https://github.com/skypilot-org/skypilot/blob/8737e87b3cbe60c72a1aae1c68d70371ad9f0dfb/tests/conftest.py#L141-L146).

Reply:
fixed